src/openms/include/OpenMS/ANALYSIS/OPENSWATH/MRMAssay.h

```diff
@@ -325,12 +338,12 @@ namespace OpenMS
       TODO: why make copy of IonMapT ?
```

Comment:
@grosenberger can you comment on whether we could change pass `DecoyIonMap` and `TargetIonMap` by reference?

Reply:
Yes, we should change that, thanks for spotting this. They are only used as reference and should not be modified.